liquid-integration/src/main/res/layout/activity_webview_quick_exchange.xml

```diff
@@ -33,7 +33,9 @@
     <ViewSwitcher
         android:id="@+id/view_switcher"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        tools:visibility="visible"
```

Comment:
These lines were removed in a previous PR by mistake and that prevented the close button from being visible.